doc/cephadm/host-management.rst

```diff
@@ -64,48 +64,37 @@ To add each new host to the cluster, perform two steps:
 Removing Hosts
 ==============
 
-If the node that want you to remove is running OSDs, make sure you remove the OSDs from the node.
+A host can safely be removed from a the cluster once all daemons are removed from it.
 
-To remove a host from a cluster, do the following:
+To drain all daemons from a host do the following:
 
-For all Ceph service types, except for ``node-exporter`` and ``crash``, remove
-the host from the placement specification file (for example, cluster.yml).
-For example, if you are removing the host named host2, remove all occurrences of
-``- host2`` from all ``placement:`` sections.
+.. prompt:: bash #
 
-Update:
+  ceph orch host drain *<host>*
 
-.. code-block:: yaml
+the '_no_schedule' label will be applied to the host. See :ref:`cephadm-special-host-labels`
 
-  service_type: rgw
-  placement:
-    hosts:
-    - host1
-    - host2
 
-To:
+all osds on the host will be scheduled to be removed. you can check osd removal progress with the follwing:
```

Comment:
```suggestion
All osds on the host will be scheduled to be removed. You can check osd removal progress with the following:
```

Also, can we also capitalize the first letter of each statement in these docs. E.g. y in "you can check . . .", t in "the _no_schedule . . ." etc. That seems to be the format we're using mostly in these docs